src/grading-settings/grading-scale/GradingScale.test.jsx

```diff
@@ -16,7 +16,7 @@ const sortedGrades = [
   { current: 20, previous: 0 },
 ];
 
-const RootWrapper = () => (
+const RootWrapper = (props = {}) => (
```

Comment:
I would pass in `viewOnly` as its own individual prop unless there are other props that will passed in.